diff --git a/bindings/bindingtester/__init__.py b/bindings/bindingtester/__init__.py
index b4daa2d..aecb623 100644
--- a/bindings/bindingtester/__init__.py
+++ b/bindings/bindingtester/__init__.py
@@ -21,7 +21,7 @@
 import sys
 import os

-sys.path[:0] = [os.path.join(os.path.dirname(__file__), '..', '..', 'bindings', 'python')]
+sys.path[:0] = ['/home/travis/.local/lib/python2.7/site-packages/']

 import util

diff --git a/bindings/bindingtester/known_testers.py b/bindings/bindingtester/known_testers.py
index b7aca68..1656819 100644
--- a/bindings/bindingtester/known_testers.py
+++ b/bindings/bindingtester/known_testers.py
@@ -63,4 +63,5 @@ testers = {
     'java_async': Tester('java', _java_cmd + 'AsyncStackTester', 2040, 510, MAX_API_VERSION, types=ALL_TYPES),
     'go': Tester('go', _absolute_path('go/build/bin/_stacktester'), 63, 200, MAX_API_VERSION),
     'flow': Tester('flow', _absolute_path('flow/bin/fdb_flow_tester'), 63, 500, MAX_API_VERSION),
+    'elixir': Tester('elixir', './test/binding_tester.sh', 2040, 510, MAX_API_VERSION, types=['null', 'bytes', 'string', 'int', 'uuid', 'bool', 'tuple', 'versionstamp'])
 }
diff --git a/bindings/bindingtester/tests/api.py b/bindings/bindingtester/tests/api.py
index cf7eb39..efad26a 100644
--- a/bindings/bindingtester/tests/api.py
+++ b/bindings/bindingtester/tests/api.py
@@ -149,7 +149,7 @@ class ApiTest(Test):
         mutations += ['VERSIONSTAMP']
         versions = ['GET_READ_VERSION', 'SET_READ_VERSION', 'GET_COMMITTED_VERSION']
         snapshot_versions = ['GET_READ_VERSION_SNAPSHOT']
-        tuples = ['TUPLE_PACK', 'TUPLE_UNPACK', 'TUPLE_RANGE', 'TUPLE_SORT', 'SUB', 'ENCODE_FLOAT', 'ENCODE_DOUBLE', 'DECODE_DOUBLE', 'DECODE_FLOAT']
+        tuples = ['TUPLE_PACK', 'TUPLE_UNPACK', 'TUPLE_RANGE', 'TUPLE_SORT', 'SUB']
         if 'versionstamp' in args.types:
             tuples.append('TUPLE_PACK_WITH_VERSIONSTAMP')
         resets = ['ON_ERROR', 'RESET', 'CANCEL']
diff --git a/bindings/python/tests/tester.py b/bindings/python/tests/tester.py
index daa16fb..f48d004 100755
--- a/bindings/python/tests/tester.py
+++ b/bindings/python/tests/tester.py
@@ -30,7 +30,7 @@ import random
 import time
 import traceback

-sys.path[:0] = [os.path.join(os.path.dirname(__file__), '..')]
+sys.path[:0] = ['/home/travis/.local/lib/python2.7/site-packages/']
 import fdb
 fdb.api_version(int(sys.argv[2]))
