diff --git a/bindings/bindingtester/__init__.py b/bindings/bindingtester/__init__.py
index 753f2202..9d1d1fab 100644
--- a/bindings/bindingtester/__init__.py
+++ b/bindings/bindingtester/__init__.py
@@ -21,7 +21,7 @@
 import sys
 import os
 
-sys.path[:0] = [os.path.join(os.path.dirname(__file__), '..', '..', 'bindings', 'python')]
+sys.path[:0] = ['USER_SITE_PATH']
 
 import util
 
diff --git a/bindings/bindingtester/known_testers.py b/bindings/bindingtester/known_testers.py
index 7606a23c..7a8f9df6 100644
--- a/bindings/bindingtester/known_testers.py
+++ b/bindings/bindingtester/known_testers.py
@@ -64,4 +64,5 @@ testers = {
     'java_async': Tester('java', _java_cmd + 'AsyncStackTester', 2040, 510, MAX_API_VERSION, types=ALL_TYPES),
     'go': Tester('go', _absolute_path('go/build/bin/_stacktester'), 2040, 200, MAX_API_VERSION),
     'flow': Tester('flow', _absolute_path('flow/bin/fdb_flow_tester'), 63, 500, MAX_API_VERSION, directory_snapshot_ops_enabled=False),
+    'elixir': Tester('elixir', './test/binding_tester.sh', 2040, 520, MAX_API_VERSION, types=ALL_TYPES)
 }
diff --git a/bindings/python/tests/tester.py b/bindings/python/tests/tester.py
index 6aefe7eb..72cd4d59 100644
--- a/bindings/python/tests/tester.py
+++ b/bindings/python/tests/tester.py
@@ -30,7 +30,7 @@ import random
 import time
 import traceback
 
-sys.path[:0] = [os.path.join(os.path.dirname(__file__), '..')]
+sys.path[:0] = ['USER_SITE_PATH']
 import fdb
 fdb.api_version(int(sys.argv[2]))
 
